frontend/packages/operator-lifecycle-manager/src/components/descriptors/spec/match-expressions.tsx

```diff
@@ -53,19 +55,24 @@ const MatchExpression: React.FC<MatchExpressionProps> = ({
         <div className="key-operator-value__heading hidden-md hidden-lg text-secondary text-uppercase">
           {t('olm~Values')}
         </div>
-        <input
-          className="pf-v5-c-form-control"
-          type="text"
-          value={values?.join(',') ?? ''}
-          onChange={(e) =>
-            onChange({
-              key,
-              operator,
-              values: e.target?.value?.split(',')?.map((v) => v.trim()) ?? [],
-            })
-          }
-          disabled={UNARY_OPERATORS.includes(operator as Operator)}
-        />
+        <span
+          className={classNames('pf-v6-c-form-control', {
+            'pf-m-disabled': UNARY_OPERATORS.includes(operator as Operator),
```

Comment:
Pull `UNARY_OPERATORS.includes(operator as Operator)` out to a `const` so it's not duplicated.